### PR TITLE
Minor Corrections

### PR DESCRIPTION
**Pull request by OshekharO**
some minor grammatical corrections

### CODE REVIEW

**Comment by OshekharO** [comment]
Can be merged

File: README.md [hunk 1]
```diff
@@ -35,18 +35,18 @@ Library that provides manga sources.
...
-   See for versions at [JitPack](https://jitpack.io/#KotatsuApp/kotatsu-parsers)
+   Versions are available on [JitPack](https://jitpack.io/#KotatsuApp/kotatsu-parsers)
 
 3. Usage in code
 
    ```kotlin
    val parser = MangaSource.MANGADEX.newParser(mangaLoaderContext)
    ```
 
-   `mangaLoaderContext` is an implementation of the `MangaLoaderContext` class.
```
Comment (Koitharu) [inline README.md:46]: Why did you remove graves around the "MangaLoaderContext" word?

**Comment by Koitharu** [comment]
Looks fine for me, @ztimms73 what do you think?

**Review by Koitharu — APPROVED** [review]

**Review by ztimms73 — APPROVED** [review]
it's okay now

**Comment by Koitharu** [comment]
Thanks
